applications/CompressiblePotentialFlowApplication/tests/cpp_tests/test_potential_flow_utilities.cpp

```diff
@@ -99,6 +173,91 @@ KRATOS_TEST_CASE_IN_SUITE(ComputeLocalMachNumber, CompressiblePotentialApplicati
     KRATOS_CHECK_NEAR(local_mach_number, 0.748948914, 1e-6);
 }
 
+// Checks ComputeLocalMachNumberSquared in utilities, local velocity that should be clamped
+KRATOS_TEST_CASE_IN_SUITE(ComputeLocalMachNumberSquared, CompressiblePotentialApplicationFastSuite) {
+    Model this_model;
+    ModelPart& model_part = this_model.CreateModelPart("Main", 3);
+
+    AssignFreeStreamValues(model_part);
+
+    double reference_local_mach_squared = 3.00000000000000000000;
```

Comment:
I don't think you need so many zeros
```suggestion
    const double reference_local_mach_squared = 3.0;
```